### PR TITLE
docs revamp: Service accounts

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

Add a description of the usage and benefits of Service account tokens.

**Why do we need this feature?**

[Design doc](https://docs.google.com/document/d/1h8-AZMnYI0FT-cTIZDUSJm0KxXpXHKvyfFQ6neg9hDA/edit#heading=h.5sybau7waq2q)

[Progress Roadmap](https://docs.google.com/spreadsheets/d/1kbT1hvWdkIVw-ZM8G6G9dbGIl09CntpQP9fo0iKwd6k/edit#gid=792824740)

**Who is this feature for?**

Everyone!

**Special notes for your reviewer**:

This PR will be merged against [a base PR](https://github.com/grafana/grafana/pull/62582) to reduce the scope of the reviewing process.

### CODE REVIEW

File: docs/sources/auth/planning/index.md [hunk 1]
```diff
@@ -85,7 +85,38 @@ Consider the following scenarios:
 
 These are just a few examples of how Grafana can be used in M2M scenarios. The platform is highly flexible and can be used in a variety of other M2M applications, making it a powerful tool for organizations looking to gain insights into their systems and devices.
 
-### 🚧 Service Accounts
+### Service accounts
+
+You can use a service account to run automated workloads in Grafana, such as dashboard provisioning, configuration, or report generation. Create service accounts and service accounts tokens to authenticate applications, such as Terraform, with the Grafana API.
+
+> **Note:** Service accounts will eventually replace [API keys]({{< relref "../api-keys/" >}}) as the primary way to authenticate applications that interact with Grafana.
```
Comment (chri2547) [inline docs/sources/auth/planning/index.md:92]: Has this been completed yet? Wondering if this is still accurate.

**Review by chri2547 — APPROVED** [review]
Just one question, looks great.

File: docs/sources/auth/planning/index.md [hunk 2]
```diff
@@ -85,7 +85,38 @@ Consider the following scenarios:
...
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control/" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+> **Note:** Service accounts can only act in the organization they are created for. If you have the same task that is needed for multiple organizations, we recommend creating service accounts in each organization.
+
+{{< vimeo 742056367 >}}
+<br>
+
+## Service account tokens
+
+A service account token is a generated random string that acts as an alternative to a password when authenticating with Grafana's HTTP API.
```
Comment (Jguer) [inline docs/sources/auth/planning/index.md:110]: prefixed with `glsa` , I feel this information is important so people can recognize a token easily

File: docs/sources/auth/planning/index.md [hunk 3]
```diff
@@ -85,7 +85,38 @@ Consider the following scenarios:
...
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control/" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+> **Note:** Service accounts can only act in the organization they are created for. If you have the same task that is needed for multiple organizations, we recommend creating service accounts in each organization.
+
+{{< vimeo 742056367 >}}
```
Comment (IevaVasiljeva) [inline docs/sources/auth/planning/index.md:105]: nit: is this a typo? Or is this an actual  video about service accounts?

**Review by IevaVasiljeva — COMMENTED** [review]
Some small comments.

**Review by Jguer — APPROVED** [review]

File: docs/sources/auth/planning/index.md (hunk 3 above)
Comment (linoman) [reply docs/sources/auth/planning/index.md:105]: That's the actual video, however I dunno how to render it locally.

Comment (linoman) [reply docs/sources/auth/planning/index.md:105]: You can render the docs locally in order to see what the video looks like.

```
cd docs
make docs
```
image

File: docs/sources/auth/planning/index.md (hunk 2 above)
Comment (linoman) [reply docs/sources/auth/planning/index.md:110]: Agreed! However, keep in mind this section is only about a brief description of Service account tokens during planning. I'll make sure to include this during the reference development for SAT

**Pull request merged**